_stylesheet.css

```diff
@@ -1,6 +1,34 @@
 /* Main Layout */
+html {
+    position: relative;
```

Comment:
I think this will lead to further problems, I see why you are using `position: relative` here - but I think is a big price to pay. We should focus on keeping the default template as clean and simple as possible - in a way that users can then easily customize it.